Removing CLI command information

Luba89 opened the pull request:
CLI command is showing a version of the GDNSL; currently, there is no command that will show the version of the platform.

Luba89 committed:
Removing CLI command information

CLI command is showing a version of the GDNSL; currently, there is no command that will show the version of the platform.

diana-macrometa reviewed (approved):
Please update this in the 17.10 release notes as well.

Luba89 committed:
Removing CLI command information

CLI command is showing a version of the GDNSL; currently, there is no command that will show the version of the platform.

Luba89 committed:
Merge branch 'master' into patch-9

jamesm-macrometa committed:
Merge branch 'master' into patch-9

diana-macrometa committed:
Merge branch 'master' into patch-9

diana-macrometa committed:
Merge branch 'master' into patch-9
